### DISCUSSION TITLE
update self hosted instances?

### DISCUSSION THREAD

**Discussion by piraterx** [simple-login/app#703] [Self-hosting question]
Hey, I am trying out self hosted version!!
Do I need to pay through upgrade for unlimited access? I tried and gives error as it doesn't resolve anywhere.
How smooth does upgrade or update process goes and how do we do when new version come up? Where do we know in webapp that there is new version?

**Comment by nguyenkims** [comment]
No you don't have to pay to have unlimited access. This is extracted from the README file:

```
By default, new accounts are not premium so don't have unlimited alias. To make your account premium,
please go to the database, table "users" and set "lifetime" column to "1" or "TRUE".
```

**Reply by piraterx** [reply]
> No you don't have to pay to have unlimited access. This is extracted from the README file:
> 
> ```
> By default, new accounts are not premium so don't have unlimited alias. To make your account premium,
> please go to the database, table "users" and set "lifetime" column to "1" or "TRUE".
> ```

Thank you for such a product. What about update process?

**Reply by nguyenkims** [reply]
When a new version is released, you can upgrade your instance by following this document https://github.com/simple-login/app/blob/master/docs/upgrade.md . The upgrade process should be smooth as we avoid breaking changes as much as possible.

We'll announce a new version on Twitter, Reddit when it's available :).

**Reply by piraterx** [reply]
You are awesome!! I will donate I promise. Budget is tight right now.

**Reply by nguyenkims** [reply]
@piraterx We have released a beta version for the self-hosted SimpleLogin. The upgrade doc is on https://github.com/simple-login/app/blob/4.6.2-beta/docs/upgrade.md
